src/components/Shared/HeaderAlert.tsx

```diff
@@ -0,0 +1,34 @@
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import { connect } from 'react-redux';
+
+import useAlert from '../../hooks/useAlert';
+import type { AlertType, State } from '../../types/state';
+
+function HeaderAlert(props: HeaderAlertProps) {
+  const { alert } = props;
+  const { useClearAlert } = useAlert();
+  return (
+    <Alert
+      severity={alert.severity}
+      onClose={() => {
```

Comment:
It would probably be a good idea to call `e.stopPropagation()` on the event passed to the callback, so that the close operation doesn't trigger other things (eg: so that it doesn't close the select box)

Reply:
thanks!